### DISCUSSION TITLE
how to  snap bind A and w while A and D work?

### DISCUSSION THREAD

**Discussion by stvnkng** [cafali/SnapKey#53] [Q&A] [answered]
how to  snap bind A and w while A and D work?

i move foward with W
move left  with A

should i recompile the c++? if yes which part in cpp should i edit? thanks

**Comment by minteeaa** [comment, marked as answer]
should be able to do this with [SK1.2.0](https://github.com/cafali/SnapKey/releases/tag/v1.2.0) - just set more keys for one group in the config
